coverage.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+forge install
+find lib/aave-v3-core/contracts -type f -name "*.sol" -exec sed -i 's/pragma solidity 0\.8\.10;/pragma solidity 0.8.13;/g' {} +
+find src/ATokenFlexVoting.sol -type f -name "*.sol" -exec sed -i 's/pragma solidity 0\.8\.10;/pragma solidity 0.8.13;/g' {} +
+find test/ATokenFlexVotingFork.t.sol -type f -name "*.sol" -exec sed -i 's/pragma solidity >=0\.8\.10;/pragma solidity >=0.8.13;/g' {} +
+forge coverage --report summary --report lcov --ir-minimum
+find ./lib/aave-v3-core/contracts -type f -name "*.sol" -exec sed -i 's/pragma solidity 0\.8\.13;/pragma solidity 0.8.10;/g' {} +
```

Comment:
For example, I'd do something like:

```sh
# First, we change all Aave pragmas from 0.8.10 to 0.8.13, because this is the first solc version with via-ir and we need via-ir because .....
find lib/aave-v3-core/contracts -type f -name "*.sol" -exec sed -i 's/pragma solidity 0\.8\.10;/pragma solidity 0.8.13;/g' {} +
# --- snip --- 

# Now we run coverage
forge coverage --report summary --report lcov --ir-minimum

# Regardless of whether coverage succeeded, we want to change the pragmas back

Reply:
https://github.com/ScopeLift/flexible-voting-clients/pull/1/commits/4dfff53f6f96accfc59fca2b31d115e0b2e161b0